CHANGELOG.md

```diff
@@ -38,15 +39,15 @@ Here is a full list of what went live with version 1 of superorgs:
 
 * SDK support for superorgs and all assuming mechanism. Both for JS and Python sdks. [See the SDK here](https://github.com/aflorithmic/apiaudio-python#super-organizations)
 
-## Billing
+### Billing
 We've optimised our pricing for usage. 
 * 250 free credits on sign up, instead of 500.
```

Comment:
```suggestion
* All new users will now receive 250 free credits on sign up, instead of 500. You will have 30 days to use this allowance and start experimenting with the API. 
```